Sources/CoreCommands/SwiftCommandState.swift

```diff
@@ -98,14 +98,14 @@ extension _SwiftCommand {
     }
 }
 
-public protocol SwiftCommand: ParsableCommand, _SwiftCommand {
-    func run(_ swiftCommandState: SwiftCommandState) throws
+public protocol SwiftCommand: AsyncParsableCommand, _SwiftCommand {
```

Comment:
How is this protocol different now from the existing `AsyncSwiftCommand`? Why that protocol is not used instead?

Reply:
I reverted this, that was overlooked from my side. Thanks for pointing that out.